exonum/src/explorer/mod.rs

```diff
@@ -55,6 +55,25 @@ pub struct TxInfo {
     pub proof_to_block_merkle_root: ListProof<Hash>,
 }
 
+/// Range Information
+#[derive(Debug, Serialize)]
+pub struct Range {
+    /// Left border of the range, >=0
+    pub from: u64,
+    /// Right border of the range
+    pub to: u64,
+}
+
+/// Information on blocks coupled with the corresponding
+/// range in the clockchain
```

Comment:
Typo: **b**lockchain.

Reply:
All right, I was not aware of the full-stop thing. 
Which release will these changes be included in?